### ISSUE TITLE
Stop backprop for some samples

### ISSUE THREAD

**Issue by a-maci** [ppwwyyxx/tensorpack#86]
Is there a way to not do the back-prop step during training?
I want to not do this step for samples that are correctly classified.

**Comment by ppwwyyxx** [comment]
You can use `tf.select(mask, cost, tf.stop_gradient(cost))` to mask out some samples.
